### PR TITLE
Update Lithops version

### PR DESCRIPTION
**Pull request by sergii-mamedov**
We are updating to avoid the problems [described here](https://github.com/metaspace2020/metaspace/issues/795).

### CODE REVIEW

**Review by LachlanStuart — APPROVED** [review]
I'm approving as the changes look fine, but please don't merge it until you've finished testing on Staging, as `master` should be kept in a "ready to deploy to production with minimal testing" state.

**Comment by sergii-mamedov** [comment]
copy that

File: metaspace/engine/docs/Lithops.md [hunk 1]
```diff
@@ -168,4 +168,4 @@ Alternatively, you can use the IBM Cloud CLI to download objects to the local fi
 ```
 ibmcloud cos objects --bucket BUCKET
 ibmcloud cos object-get --bucket BUCKET --key KEY DEST_FILENAME
-```
```
Comment (LachlanStuart) [inline metaspace/engine/docs/Lithops.md:171]: BTW I've moved the `Lithops.md` from this branch into the wiki here: https://github.com/metaspace2020/metaspace/wiki/Lithops-cloud-infrastructure-setup

If you make any more changes, please do them in the wiki as I'll soon delete this file from master branch.

Comment (sergii-mamedov) [reply metaspace/engine/docs/Lithops.md:171]: got it

**Comment by LachlanStuart** [comment]
Retroactive LGTM :+1: (we agreed it was fine to merge on Slack)

**Pull request merged**